src/components/transactions/votedDelegates.js

```diff
@@ -17,19 +17,25 @@ class VotedDelegates extends AccountList {
           <div className={styles.label}>
             <div className='voters-value'>
               {this.props.t('Who voted for this delegate')}
-              {` (${this.state.votersSize})`}
+              {` (${this.props.votersSize || 0})`}
             </div>
-            {super.renderSearchFilter('votersFilterQuery', t('Filter voters'))}
           </div>
           <div className={styles.value}>
             {voters && voters
-              .slice(0, this.state.showVotersNumber)}
+                .slice(0, this.state.showVotersNumber)}
           </div>
           {voters.length > this.state.showVotersNumber ?
             <div onClick={() => { super.showMore('showVotersNumber'); }} className={`${styles.showMore} showMore  show-voters`}>
               <FontIcon className={styles.arrowDown} value='arrow-down'/>
               {this.props.t('Show more')}
-            </div> : ''
+            </div> : <Waypoint
```

Comment:
IMO it's easier to read if there is a component in the ternary when it starts on its own line.
```suggestion
            </div> :
            <Waypoint
```